x-pack/plugins/secops/public/components/flyout/button/index.tsx

```diff
@@ -76,7 +76,11 @@ export const FlyoutButton = pure(
           droppableId={`${droppableTimelineFlyoutButtonPrefix}${timelineId}`}
           theme={theme}
         >
-          <BadgeButtonContainer data-test-subj="flyoutOverlay" onClick={onOpen}>
+          <BadgeButtonContainer
+            className="flyout-overlay"
```

Comment:
that's here because in droppable-wrapper.ts, I need to overwrite stuff outside of this component when it is DraggingOver. I did not see another way of doing that with styled components without implicating a lot of props. When we will move to React 16.6.0, we can use context and that will be much cleaner. I think that's the simpler way to implement for now.

Reply:
Also, we are using the extending style from styled-component for the EUI components, but sometimes we need to go deeper and change children HTML element with classes to get the result that we want.